bittensor/subtensor.py

```diff
@@ -4935,9 +5136,9 @@ def make_substrate_call_with_retry():
             return self.substrate.query_map(
                 module="System",
                 storage_function="Account",
-                block_hash=(
-                    None if block is None else self.substrate.get_block_hash(block)
-                ),
+                block_hash=None
+                if block is None
+                else self.substrate.get_block_hash(block),
```

Comment:
Can we put these in parentheses like:
```python
block_hash=(
    None if block is None else self.substrate.get_block_hash(block)
),
```

I know it's a black formatting issue, but this is really hard to read at first